nordic/trezor/README.md

```diff
@@ -31,21 +31,22 @@ Follow these steps to set up the project on your local machine.
 
 Using nrfutil, install the required toolchain for the nRF Connect SDK:
 ```sh
-nrfutil toolchain-manager install --ncs-version v2.6.2
+nrfutil toolchain-manager install --ncs-version v2.9.0
 ```
 
 ### Launch the nRF Shell
 
-First, launch the nRF shell using the `nrfutil` toolchain manager:
+First, launch the nRF shell using the `nrfutil` toolchain manager and set the NCS to chosen version:
 
 ```sh
 nrfutil toolchain-manager launch --shell
+west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.9.0 v2.9.0
```

Comment:
i don''t think this is needed; we have west init below which will use the west.yml already specifying 2.9.0, so that, along with west update will do the trick

Reply:
I am really unsure about this, it should be as you say, but I had to rebuild fresh many times until pythons from 2 toolchains stopped interfering with each other (it seemed without using this it mixed selected from `west.yml` with toolchain `nrfutil toolchain-manager list` listed as default).

Reply:
I changed it to say to use `--pristine always` on first build.